Analytical_Engine/Create/Graph/Graph.cs

```diff
@@ -129,6 +129,12 @@ public static Graph Graph<T>(List<ICurve> connectingCurves, T prototypeEntity, L
             if (entities == null)
                 entities = new List<IElement0D>();
 
+            if (!entities.All(e => e is IBHoMObject && e is IGeometry))
```

Comment:
This would now exclude objects which were of type [INode](https://github.com/BHoM/BHoM/blob/develop/Analytical_oM/Elements/INode.cs) from working with the graphing capabilities of Analytical_Engine, which is fairly key for this engine to remain capable of working on objects from its own linked oM in my opinion.